### PR TITLE
[APPSEC-10347] Add necessary code for automated login events in  extended mode for ruby

### PR DESCRIPTION
**Pull request by GustavoCaso**
## Description

Add the necessary information on the automated login event in the extended mode for ruby. 

The tests are not yet enabled as the feature has not being release. The test have been tested locally 😄 

## Workflow

1. ⚠️⚠️ Create your PR as draft
2. Follow the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
3. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
4. Mark it as ready for review

Once your PR is reviewed, you can merge it! :heart:

## Reviewer checklist

* [ ] Check what scenarios are modified. If needed, add the relevant label (`run-parametric-scenario`, `run-profiling-scenario`...). If this PR modifies any system-tests internal, then add the `run-all-scenarios` label ([more info](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md)). 
* [ ] CI is green
   * [ ] If not, failing jobs are not related to this change (and you are 100% sure about this statement)
* if any of `build-some-image` label is present
  1. is the image labl have been updated ? 
  2. just before merging, locally build and push the image to hub.docker.com

### CODE REVIEW

**Review by anna-git — APPROVED** [review]

File: tests/appsec/test_automated_login_events.py [hunk 1]
```diff
@@ -326,18 +336,15 @@ def test_login_success_basic(self):
             assert meta["_dd.appsec.events.users.login.success.auto.mode"] == "extended"
             assert meta["appsec.events.users.login.success.track"] == "true"
             assert meta["usr.id"] == "social-security-id"
-
-            if context.library != "php":
-                assert meta["usr.email"] == "testuser@ddog.com"
```
Comment (GustavoCaso) [inline tests/appsec/test_automated_login_events.py:331]: No need to mention php, since this feature is missing on their implementation

**Comment by hoolioh** [comment]
Add run-all-scenarios label so the extended tests are executed.

File: tests/appsec/test_automated_login_events.py [hunk 2]
```diff
@@ -384,31 +391,37 @@ def test_login_wrong_user_failure_basic(self):
...
-            "/login?auth=local", data={self.username_key: self.USER, "password": "12345"}
+            "/login?auth=local", data={self.username_key: self.USER, self.password_key: "12345"}
         )
 
     def test_login_wrong_password_failure_local(self):
         assert self.r_wrong_user_failure.status_code == 401
         for _, _, span in interfaces.library.get_spans(request=self.r_wrong_user_failure):
             meta = span.get("meta", {})
-            if context.library != "nodejs":
+            if context.library == "nodejs":
```
Comment (cbeauchesne) [inline tests/appsec/test_automated_login_events.py:416]: Typo ?

Comment (cbeauchesne) [reply tests/appsec/test_automated_login_events.py:416]: oh, sorry, I missed the inversion of the `if`/`then` contents ...

**Pull request merged**